app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json

```diff
@@ -107,6 +107,8 @@
   "permissionSelectAll": "Select all ({{allListLength}} items)",
   "permissionSet": "Set permissions",
   "permissionSeletedViews": "Grant or Revoke permissions for the selected Views: ",
+  "permissionShowLess": "Show less",
```

Comment:
these can be moved to shared and can remove permission from the key.

Reply:
agree - there are probably others that could be in shared too